### PR TITLE
[Hotfix] Fix Import token modal blinking on network switching

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

image

Import token modal is blinking on network switching

  # To Test

1. Connect wallet to Ethereum Mainnet
2. Switch network to Gnosis Chain from UI
**ER:** UI updated without opening the Import token modal
**AR:** Import token modal appears for ~1sec and disappears

### CODE REVIEW

File: src/cow-react/modules/swap/containers/ImportTokenModal/index.tsx [hunk 1]
```diff
@@ -46,10 +46,16 @@ export function ImportTokenModal(props: ImportTokenModalProps) {
           // Any token addresses that are loaded from the shorthands map do not need to show the import URL
           const supported = supportedChainId(chainId)
           if (!supported) return true
```
Comment (alfetopito) [inline src/cow-react/modules/swap/containers/ImportTokenModal/index.tsx:48]: Not part of this PR, I know, but this check doesn't need to be done inside the loop as it won't change with the token

**Review by alfetopito — APPROVED** [review]
Great fix, working very nicely!

Just one nitpick that's not even related to the current code

**Review by elena-zh — APPROVED** [review]
Hey @shoom3301 , changes look great!

File: src/cow-react/modules/swap/containers/ImportTokenModal/index.tsx (hunk 1 above)
Comment (shoom3301) [reply src/cow-react/modules/swap/containers/ImportTokenModal/index.tsx:48]: I agree, it looks weird, but the check is located inside of `filter` and the function returns an array of tokens.
The file should be refactored, I will add TODO
